ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.22__4.10.yaml

```diff
@@ -92,7 +92,7 @@ images:
   to: serverless-stop-bundle
 promotion:
   name: openshift-serverless-v1.22.0
-  namespace: openshift
```

Comment:
is this for promotion on the internal registry, not sure we did set some namespace before.

A bit puzzled on this

Reply:
+1 How was this previously ok? Did something change?

Reply:
It will be used in https://github.com/openshift-knative/serverless-operator/pull/1569

Reply:
Taking another look, the `openshift` namespace is not recommended but the images are published there anyway. Checked by `skopeo inspect docker://registry.ci.openshift.org/openshift/openshift-serverless-nightly:serverless-bundle` and the image was created yesterday. 
Do you want to change it back or can we proceed with `knative` namespace?

Reply:
IMO. It would be good to fix the warning and have a clean run.